src/overlays/actors/ovl_Bg_Dkjail_Ivy/z_bg_dkjail_ivy.c

```diff
@@ -28,47 +36,177 @@ const ActorInit Bg_Dkjail_Ivy_InitVars = {
     (ActorFunc)BgDkjailIvy_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80ADE950 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_NONE, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x01000200, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x01000200, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
     { 20, 80, 0, { 0, 0, 0 } },
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80ADE98C[] = {
+static Gfx* sLeafDlists[] = { gKakeraLeafMiddle, gKakeraLeafTip };
+
+static s16 sLeafScales[] = { 110, 80, 60, 40 };
```

Comment:
Looks like these can be in-function static if you move the initchain to above Init.